core/arch/arm/kernel/abort.c

```diff
@@ -123,6 +124,37 @@ static __maybe_unused const char *fault_to_str(uint32_t abort_type,
 	}
 }
 
+#ifdef CFG_CORE_SYMS
+/*
+ * Helper function this returns formated symbol name (if any) from a static
+ * storage. This is to ease up printing of symbols.
+ *
+ * Do not try to call this function multiple times for the same format
+ * string. I.e. do NOT do this:
+ * EMSG("%s %s", get_sym_name(v1, c), get_sym_name(v2, c));
+ */
+static __maybe_unused const char *get_sym_name(vaddr_t va, size_t core_pos)
+{
+	/* In case we'll have abort on several cores at the same time  */
+	static char sym_name[CFG_TEE_CORE_NB_CORE][64];
```

Comment:
init `= {};`  and `= 0;` below.

Double space after `same time  */` in inline comment.